modules/end-user-guide/partials/proc_overriding-devfile-values-using-factory-parameters.adoc

```diff
@@ -37,7 +37,7 @@ To add or override `generateName` value, use the following factory URL:
 
 [subs="+quotes"]
 ----
-https://che.openshift.io/f?url=https://gist.githubusercontent.com/themr0c/ef8e59a162748a8be07e900b6401e6a8/raw/8802c20743cde712bbc822521463359a60d1f7a9/devfile.yaml&override.metadata.generateName=myprefix
+https://workspaces.openshift.com/f?url=https://gist.githubusercontent.com/themr0c/ef8e59a162748a8be07e900b6401e6a8/raw/8802c20743cde712bbc822521463359a60d1f7a9/devfile.yaml&override.metadata.generateName=myprefix
```

Comment:
@themr0c why are you using the personal raw devfiles in the samples?

Reply:
That's probably a leftover...

Reply:
```suggestion
https://workspaces.openshift.com/f?url=https://github.com/eclipse/che-docs&override.metadata.generateName=myprefix
```